src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -429,6 +448,52 @@ export default class ControlledTable extends PureComponent<
         return document.getElementById(this.props.id) as HTMLElement;
     }
 
+    private containsActiveElement(): boolean {
+        const $el = this.$el;
+
+        return $el && $el.contains(document.activeElement);
+    }
+
+    private getActiveCellAttributes(): {
+        column_id: string | null;
+        row: number | null;
+    } | void {
+        let activeElement = document.activeElement;
+        while (activeElement && activeElement.nodeName.toLowerCase() !== 'td') {
+            activeElement = activeElement.parentElement;
+        }
+
+        if (!activeElement) {
+            return;
+        }
+
+        const column_id = activeElement.getAttribute('data-dash-column');
+        const row = activeElement.getAttribute('data-dash-row');
+
+        return {column_id, row: +(row ?? 0)};
+    }
+
+    /*#if TEST_COPY_PASTE*/
+    private preventCopyPaste(): boolean {
+        if (!this.containsActiveElement()) {
+            return false;
+        }
+
+        const {active_cell} = this.props;
+        const active = this.getActiveCellAttributes();
+
+        if (
+            !active ||
+            active.column_id !== active_cell?.column_id ||
+            active.row !== active_cell?.row
+        ) {
+            return true;
+        }
+
+        return false;
+    }
```

Comment:
I am unable to use Ctrl+C/Ctrl+V in a way that triggers copy/paste in Selenium, Chrome does not support `document.execCommand('paste')` and the ClipboardAPI requires the user to `Allow` browser permission for usage. These reasons are behind the special "test" build and also behind this long standing regression. Prior to this change, the tests worked but the real build didn't. This tries to better align the test behavior vs. the real world behavior.